src/xn.ml

```diff
@@ -291,6 +291,8 @@ let parse_vif vm_id (x, idx) =
 		other_config = [];
 		locking_mode = Vif.Unlocked;
 		extra_private_keys = [];
+		ipv4_configuration = Vif.Static4 (["192.168.1.100/24"; "192.168.1.200/24"], (Some "192.168.1.1"));
+		ipv6_configuration = Vif.Static6 (["2001:0DB8:02de:1000::0e13/64"], (Some "2001:0DB8:02de:1000::1"));
```

Comment:
These are a little odd as defaults... However, after adding the default records (see xcp-idl PR), we can use them: https://github.com/robhoes/xenops-cli/commit/8571ed2c6c50b43e85db43cd1c03203daf5aacd7.